nlp_primitives/lsa.py

```diff
@@ -49,18 +47,10 @@ class LSA(TransformPrimitive):
     default_value = 0
 
     def __init__(self):
-        # TODO: allow user to use own corpus
         self.number_output_features = 2
         self.n = 2
 
-        try:
-            brown = nltk.corpus.brown.sents()
-        except LookupError:
-            nltk.download('brown')
-            brown = nltk.corpus.brown.sents()
-        finally:
-            self.trainer = make_pipeline(TfidfVectorizer(), TruncatedSVD())
-            self.trainer.fit([" ".join(sent) for sent in brown])
+        self.trainer = make_pipeline(TfidfVectorizer(), TruncatedSVD(random_state=42))
```

Comment:
Might it be worth allowing the users to input their own random state as an optional argument with a default of 42?